spring/integration/pubsub/src/main/java/com/example/spring/pubsub/PubsubApplication.java

```diff
@@ -0,0 +1,153 @@
+package com.example.spring.pubsub;
+
+import com.google.pubsub.v1.Subscription;
+import com.google.pubsub.v1.SubscriptionName;
+import com.google.pubsub.v1.Topic;
+import com.google.pubsub.v1.TopicName;
+import java.io.IOException;
+
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.protobuf.ByteString;
+
+import java.util.List;
```

Comment:
Please use [Google Java Formatter](https://github.com/google/google-java-format)  for the source files.